test/gt.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { filter, gt, pipe, toArray } from "../src/index";
+
+describe("gt(greater)", function () {
+  describe("gt given curry", function () {
+    it("given array then should return greater values", function () {
+      const result = pipe([4, 5, 6], filter(gt(5)), toArray);
+      expect(result).toEqual([6]);
+    });
+    it("given array then should return empty array", function () {
+      const result = pipe([5, 6, 7], filter(gt(7)), toArray);
+      expect(result).toEqual([]);
+    });
+
+    it("given string array then should return [c, d, e]", function () {
+      const result = pipe(["a", "b", "c", "d", "e"], filter(gt("c")), toArray);
+      expect(result).toEqual(["d", "e"]);
+    });
+    it("given string array then should return empty array", function () {
+      const result = pipe(["a", "b", "c", "d"], filter(gt("d")), toArray);
+      expect(result).toEqual([]);
+    });
+  });
+
+  describe("gt given boolean", function () {
+    it("given two number should return true", function () {
```

Comment:
How about writing the testcase in the following pattern?
`should return true that first argument is greater than second argument`

Reply:
I didn't know that.


Reference was made to the test case name of 'isArray.spec.ts'.
However, the other test case names are different.

Now that I know it, I will revise it.

thx, :D